docs/architecture/full-refresh.md

```diff
@@ -0,0 +1,75 @@
+# Full Refresh
+
+This readme describes Airbyte conventions around the "full refresh" concept. Out in the world, there are many ways to define this term. We want the behavior of Airbyte integrations to be predictable, so we are adopting a preferred definition. This readme also describes what behavior to fall back on if the preferred convention cannot be used.
+
+On the n<sup>th</sup> sync of a full refresh connection:
+
+### PREFERRED: _Replace_ existing data with new data. The connection does not create any new tables.
+
+data in the destination _before_ the sync:
+
+| Languages  | 
+|---|
+| Python  |
+| Java  |
+
+new data:
```

Comment:
As a user I think I would be interested to understand what happens if data gets deleted.